README.md

```diff
@@ -27,6 +27,12 @@ Demonstrate calling the method with an argument of "young prince".
 Write your code here:
 ```ruby
 # code here
+def offerRose person do
+  return "would you take this rose and help out an old beggar,#{person}".
+end
+
+offerRose ("young prince");
```

Comment:
We generally don't put spaces between the method and the arguments in ruby, it should just be 

`offerRose("young prince")`

Also, remember in ruby we use snake_case, not camelCase for method and var names.